deadline_/challenges/tests/test_categories.py

```diff
@@ -29,7 +29,7 @@ def test_relationships(self):
 
     def test_serialize(self):
         """ the Category should show all its subcategories """
-        expected_json = '{"name":"Test","sub_categories":["Unit","Mock","Patch"]}'
+        expected_json = '{"id":1,"name":"Test","sub_categories":["Unit","Mock","Patch"]}'
```

Comment:
Change `"id": 1` to `"id": self.c1.id`
Since we moved to PostgreSQL, we are not always guaranteed that the ID of the created category will be 1. I found this out by fixing some broken tests